monai/transforms/transforms.py

```diff
@@ -145,3 +149,66 @@ def __call__(self, img):
         all_pad_width = [(0, 0), (0, 0)] + data_pad_width
         img = np.pad(img, all_pad_width, self.mode)
         return img
+
+
+@export
+class Rotate90:
+    """
+    Rotate an array by 90 degrees in the plane specified by `axes`.
+    """
+
+    def __init__(self, k=1, axes=(1, 2)):
+        """
+        Args:
+            k (int): number of times to rotate by 90 degrees.
+            axes (2 ints): defines the plane to rotate with 2 axes.
+        """
+        self.k = k
+        self.plane_axes = axes
+
+    def __call__(self, img):
+        return np.rot90(img, self.k, self.plane_axes)
+
+
+@export
+class RandRotate90(Randomizable):
+    """
+    With probability `prob`, input arrays are rotated by 90 degrees
+    in the plane specified by `axes`.
+    """
+
+    def __init__(self, prob=0.1, max_k=3, axes=(1, 2)):
+        """
+        Args:
+            prob (float): probability of rotating.
+                (Default 0.1, with 10% probability it returns a rotated array)
+            max_k (int): number of rotations will be sampled from `np.random.randint(max_k) + 1`.
+                (Default 3)
+            axes (2 ints): defines the plane to rotate with 2 axes.
+                (Default (1, 2))
+        """
+        self.prob = min(max(prob, 0.0), 1.0)
+        self.max_k = max_k
+        self.axes = axes
+
+        self._do_transform = False
+        self._rand_k = 0
+
+    def randomise(self):
+        self._rand_k = self.R.randint(self.max_k) + 1
+        self._do_transform = self.R.random() < self.prob
+
+    def __call__(self, img):
+        self.randomise()
+        if not self._do_transform:
+            return img
+        return Rotate90(self._rand_k, self.axes)(img)
```

Comment:
This can be similarly modified to store the object in a variable first. I feel it's better style even if it doesn't make a difference to avoid constructs of the form `create_something(args)(args)`.